conf/layer.conf

```diff
@@ -16,8 +16,52 @@ DISTRO_FEATURES_remove = " x11"
 DISTRO_FEATURES_remove = " kde"
 DISTRO_FEATURES_append = " egl"
 DISTRO_FEATURES_append = " opengles2"
+DISTRO_FEATURES_append = " enable_icrypto_openssl"
+DISTRO_FEATURES_append = " netflix_1080p_graphics"
+#DISTRO_FEATURES_append = " cobalt_enable_evergreen_lite"
+DISTRO_FEATURES_append = " cobalt_rialto"
+DISTRO_FEATURES_remove = " bluetooth"
 
 PACKAGECONFIG_remove_pn-cairo = " opengl"
 PACKAGECONFIG_remove_pn-qtbase = " gbm"
 
+PACKAGECONFIG_remove_pn-gstreamer1.0-plugins-bad = "vulkan"
+
+DISTRO_FEATURES_append = " cleanup_gfx"
+PREFERRED_PROVIDER_virtual/egl = "${@bb.utils.contains('DISTRO_FEATURES', 'cleanup_gfx', 'libglvnd', 'mesa', d)}"
+PREFERRED_PROVIDER_virtual/libgles1 = "${@bb.utils.contains('DISTRO_FEATURES', 'cleanup_gfx', 'libglvnd', 'mesa', d)}"
+PREFERRED_PROVIDER_virtual/libgles2 = "${@bb.utils.contains('DISTRO_FEATURES', 'cleanup_gfx', 'libglvnd', 'mesa', d)}"
+PREFERRED_PROVIDER_virtual/libgl = "${@bb.utils.contains('DISTRO_FEATURES', 'cleanup_gfx', 'libglvnd', 'mesa', d)}"
+PREFERRED_PROVIDER_virtual/mesa = "${@bb.utils.contains('DISTRO_FEATURES', 'cleanup_gfx', 'libglvnd', 'mesa', d)}"
+
+PREFERRED_PROVIDER_virtual/wpebackend = "wpebackend-rdk"
 PREFERRED_VERSION_libepoxy = "1.5.3"
+
+RDK_GIT_PROTOCOL = "https"
+CMF_GIT_PROTOCOL = "https"
+CMF_GIT_BRANCH = "rdk-next"
+CMF_GIT_ROOT = "git://code.rdkcentral.com/r"
+CMF_GITHUB_MASTER_BRANCH ?= "master"
+CMF_GITHUB_ROOT ?= "git://github.com/rdkcentral"
+RDK_RELEASE ?= "1.99"
+LICENSE_FLAGS_WHITELIST += "commercial"
```

Comment:
What is being enabled here?  In rdk.conf (meta-rdk), this is selectively enabled for particular packages.